.palantir/revapi.yml

```diff
@@ -27,6 +27,12 @@ acceptedBreaks:
     - code: "java.method.addedToInterface"
       new: "method ThisT org.apache.iceberg.SnapshotUpdate<ThisT>::scanManifestsWith(java.util.concurrent.ExecutorService)"
       justification: "Accept all changes prior to introducing API compatibility checks"
+    - code: "java.method.addedToInterface"
+      new: "method org.apache.iceberg.ContentFile<?> org.apache.iceberg.FileScanTask::contentFile()"
+      justification: "New method added to FileScanTask.  Should be not many implementations\
+        \ outside Iceberg, if so it's easy to fill in new method (especially if they\
+        \ inherit BaseFileScanTask).  This is easiest way to add DeleteFileScanTask,\
+        \ otherwise even bigger interface change is necessary"
```

Comment:
Side note: I'm looking for a way to suppress `addedToInterface` in addition to a few other non-breaking changes.

The way we release Iceberg, it's not really something that we need to worry about much.

Reply:
Yea I didn't know the new policy, thanks, good to know.   (was thinking to try adding a default too).